samples/snippets/test_get_operation.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import google.auth
+
+import pytest
+
+import get_operation
+
+
+@pytest.fixture
+def project_id():
+    _, project_id = google.auth.default()
+    return project_id
+
+
+def test_get_operation(capsys, project_id):
+    # TODO(developer): Replace this variable with your operation name.
+    operation_name = f"projects/{project_id}/locations/us-central1/operations/12345"
```

Comment:
It would be better to test against a real operation.  We have Python Samples office hours tomorrow if you'd like to come!  10AM-12PM PT.

Reply:
Completely agree! Il stop by first-thing!

Reply:
Thanks Dina. I updated the test to build and test against a real operation. PTAL!